### PR TITLE
Add features in subscription table

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds features to the table here:
https://help.sumologic.com/docs/manage/manage-subscription/cloud-flex-credits-accounts/#features-by-subscription-type

It also cleans up this article:
https://help.sumologic.com/docs/manage/manage-subscription/cloud-flex-accounts/

Issue number: #3165 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
@kimsauce, could you give this one a review? There's no rush. I just want to make sure it doesn't get lost in the shuffle.

File: docs/manage/manage-subscription/cloud-flex-accounts.md [hunk 1]
```diff
@@ -10,23 +10,20 @@ Sumo Logic provides flexible account types within its Cloud Flex packaging for
 ## Cloud Flex - Account Types
 
 * **Free** accounts give you access to core Sumo Logic features, with a daily 500 MB data volume limit. Free accounts allow three users, with 4 GB of retention. For information about Free account features, see the "Free Accounts" column in the following table and the Important notes section on [Sumo Logic Free accounts](#important-notes-on-sumo-logic-free-accounts).
-
-    :::tip
-    Free account holders can [upgrade](upgrade-cloud-flex-account.md) to Professional from within Sumo Logic.
-    :::
+:::tip
+Free account holders can [upgrade](upgrade-cloud-flex-account.md) to Professional from within Sumo Logic.
```
Comment (kimsauce) [inline docs/manage/manage-subscription/cloud-flex-accounts.md:14]:
```suggestion
Free account holders can [upgrade](upgrade-cloud-flex-account.md) to a Professional account from within Sumo Logic.
```

File: docs/manage/manage-subscription/cloud-flex-accounts.md [hunk 2]
```diff
@@ -10,23 +10,20 @@ Sumo Logic provides flexible account types within its Cloud Flex packaging for
...
+:::
 
 * **Trial** accounts allow you to try all of Sumo Logic's advanced features to understand how Sumo Logic will fit within your organization before you buy. It includes a daily data volume limit of 1 GB per day, with 20 users and 30 days of data retention. For information about Trial account features, see the "Trial Accounts" column in the table below and [Important notes on Sumo Logic Trial accounts](#important-notes-on-sumo-logic-free-accounts).  
-
-    :::note
-    Trials are limited to 30 days. If you require an extended trial period, contact [sales@sumologic.com](mailto:sales@sumologic.com) to inquire about a Proof of Concept (PoC).
-    :::
+:::note
+Trials are limited to 30 days. If you require an extended trial period, contact [sales@sumologic.com](mailto:sales@sumologic.com) to inquire about a Proof of Concept (PoC).
```
Comment (kimsauce) [inline docs/manage/manage-subscription/cloud-flex-accounts.md:19]:
```suggestion
Trials are limited to 30 days. If you need to extend your trial period, contact [sales@sumologic.com](mailto:sales@sumologic.com) to inquire about a Proof of Concept (PoC).
```

File: docs/manage/manage-subscription/cloud-flex-accounts.md [hunk 3]
```diff
@@ -134,17 +131,13 @@ To view the Account page, do the following:
 
 1. Log in to your Sumo Logic Cloud Flex account.
 1. In the left navigation bar of the UI, select **Administration** > **Account**. The Account Overview tab of the Account page is shown by default.
-
-    :::note
-    You must have a role that grants you the Account Overview capability to view the Account Overview tab.
-    :::
-
+:::note
+You must have a role that grants you the Account Overview capability to view the Account Overview tab.
```
Comment (kimsauce) [inline docs/manage/manage-subscription/cloud-flex-accounts.md:135]: I'd maybe link to the /manage capabilities doc where it explains them

File: docs/manage/manage-subscription/cloud-flex-accounts.md [hunk 4]
```diff
@@ -172,16 +165,10 @@ The following visual indicators apply:
 
 To switch between views and time interval displays, do the following:
 
-1. Log in to Sumo Logic and select **Administration** > **Account** in the left navigation pane. The Account page appears with the Account Overview tab shown by default. The top panel shows account details and the bottom panel displays usage analytics.
-
-![CloudFlex-AccountPage.png](/img/subscriptions/pqs.png)
-
+1. Log in to Sumo Logic and select **Administration** > **Account** in the left navigation pane. The Account page appears with the Account Overview tab shown by default. The top panel shows account details and the bottom panel displays usage analytics. <br/>![CloudFlex-AccountPage.png](/img/subscriptions/pqs.png)
```
Comment (kimsauce) [inline docs/manage/manage-subscription/cloud-flex-accounts.md:168]:
```suggestion
1. Log in to Sumo Logic. From the left navigation pane, select **Administration** > **Account**. The Account page appears with the Account Overview tab shown by default. The top panel shows account details and the bottom panel displays usage analytics. <br/>![CloudFlex-AccountPage.png](/img/subscriptions/pqs.png)
```

**Review by kimsauce — APPROVED** [review]
I have a few suggestions, but approving it!

File: docs/manage/manage-subscription/cloud-flex-accounts.md (hunk 3 above)
Comment (jpipkin1) [reply docs/manage/manage-subscription/cloud-flex-accounts.md]: Done.

**Comment by jpipkin1** [comment]
Thanks @kimsauce! Your suggestions were out of scope for the kind of cleanup I did on that article, but I'm glad you suggested the extra things to fix.

**Pull request merged**